package.json

```diff
@@ -97,7 +96,7 @@
     "remark-lint": "^5.0.0",
     "sass-lint": "^1.7.0",
     "seamless-immutable": "6.3.0",
-    "svg4everybody": "2.1.0",
+    "svg4everybody": "^2.1.0",
```

Comment:
Please remove this as `ember-frost-core` now provides this via the `vendor` directory.